src/annotator/util/buckets.ts

```diff
@@ -191,7 +191,7 @@ export function computeBuckets(
   // Add an upper "navigation" bucket with offscreen-above anchors
   const above: Bucket = {
     anchors: aboveAnchors,
-    position: vMargin,
+    position: vMargin - 5,
```

Comment:
Previously there was an implicit negative "top margin" due to the size of the padding. Now reducing the distance here by 5px, but without affecting the spacing between buckets or the bottom of the buckets bar, which is kept as before.